### ISSUE TITLE
Feature Request: Generate TS code in ProtoFieldName instead of lowerCamelCase

### ISSUE THREAD

**Issue by xuzaixian** [timostamm/protobuf-ts#193]
Now I have a proto like this.
```
message Request {
    int64 command_id = 1;
}
```
Now we can generate typescript interface by protoc like this.
```
export interface Request {
    commandId: bigint;
}
```
And I wonder if we can generate interface key just like in proto, not by lowerCamelCase.
This is what I expect
```
export interface Request {
    command_id: bigint;
}
```

**Comment by xuzaixian** [comment]
emmm... I'd like to have a try. PR welcomed ?

**Comment by jcready** [comment]
This has been addressed in https://github.com/timostamm/protobuf-ts/pull/250 which was released in [v2.3.0](https://github.com/timostamm/protobuf-ts/releases/tag/v2.3.0).
